src/deserialize.zig

```diff
@@ -364,3 +367,18 @@ test "detects an invalid length serialization" {
     const rlp_bytes = @embedFile("testdata/faulty_shanghai_block.rlp");
     _ = try expectError(error.InvalidSerializedLength, deserialize(Block, rlp_bytes[0..], &b));
 }
+
+test "access list empty" {
+    const StrippedTxn = struct {
+        access_list: []struct {
+            address: [20]u8,
+            storage_keys: [][32]u8,
+        },
+    };
+
+    var buf: [128]u8 = undefined;
+    const rlp = try std.fmt.hexToBytes(&buf, "c1c0");
+
+    var out: StrippedTxn = undefined;
+    _ = try deserialize(StrippedTxn, rlp, &out);
+}
```

Comment:
Test that failed (before) and passes (after).

It's a narrowed-down version on how I found this bug. Basically, an empty access list.